integration_test/third_party_apps_test/applications/apache/centos_rhel/install

```diff
@@ -1,6 +1,18 @@
 set -e
 
 sudo yum -y install httpd
-(cd /etc/httpd/conf.d/ && sudo curl -O https://raw.githubusercontent.com/Stackdriver/stackdriver-agent-service-configs/master/etc/httpd/conf.d/status.conf)
+sudo mkdir -p /etc/httpd/conf.d/
+sudo tee /etc/httpd/conf.d/status.conf >/dev/null <<EOF
+ExtendedStatus on
+<VirtualHost 127.0.0.1:80>
+    <Location /server-status>
+        SetHandler server-status
+        Order deny,allow
+        Deny from all
+        Allow from 127.0.0.1
+    </Location>
+</VirtualHost>
+EOF
 
 sudo service httpd start
+curl http://localhost:80/server-status?auto
```

Comment:
are these just for debugging? or are they intended to be a permanent change? Maybe move it into the exercise script? (integration_test/third_party_apps_test/applications/apache/exercise)

Reply:
Yes, sorry, these were for debugging the failures. Reverted.

Reply:
FWIW, I just realized that the OTel collector's `apache` receiver does not complain if the `mod_status` endpoint is not accessible, so it just silently fails to ingest the metrics (somewhat puzzling, since the `apache` receiver [ought to log these failures](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/apachereceiver/scraper.go#L67)). Adding such a `curl` would have exposed the issue in the logs. WDYT, worth bringing it back?

Note that it doesn't really belong in `exercise`, as it's not required for the test to complete — it's just a verification that installation and setup succeeded, similar to calling `nvidia_smi` in the `nvml` setup.

Reply:
spoke offline; yeah having a deubugging curl here makes sense.